mods/d2k/rules/structures.yaml

```diff
@@ -593,7 +593,7 @@ large_gun_turret:
 		QuantizedFacings: 32
 	Armament:
 		Weapon: TowerMissile
-		LocalOffset: 256,384,768, 256,-384,768
+		LocalOffset: 256,0,768
```

Comment:
What is the reason for this? This looks like it is removing some polish.
Same for the missile tank.